crates/fluvio/src/consumer/stream.rs

```diff
@@ -175,28 +213,65 @@ impl<T: Stream<Item = Result<Record, ErrorCode>> + Unpin> Stream
 }
 
 impl OffsetManagement {
+    async fn start_background_flusher(&self) {
+        match self {
```

Comment:
shouldn't this be part of flush? why it is part of retry?

Reply:
could add this as part of flush configuration

Reply:
Part of retry? This is in the initialization of `SinglePartitionConsumerStream` that is used on all ConsumerStreams

Reply:
do you mean a flag to enable it?

Reply:
trying to understand reasons for start background  worker here.  Shouldn't this be part of flush handler?